### DISCUSSION TITLE
Create Project returns error.

### DISCUSSION THREAD

**Discussion by markief1** [prisma/prisma#10597] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/95639799/144883079-7bb515ce-147e-4dcb-bf06-2488a2f8b8cb.png)
Details: [click here](https://github.com/prisma/prisma/issues/10596)

**Comment by janpio** [comment, marked as answer]
Answered in issue: https://github.com/prisma/prisma/issues/10596

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
